package.json

```diff
@@ -166,7 +166,9 @@
     "mysql2": "^3.2.3",
     "nats": "^2.11.0",
     "nats-v1": "npm:nats@^1.4.12",
+    "node-abi": "^3.54.0",
     "node-fetch": "^2.6.1",
+    "node-gyp": "^10.0.1",
```

Comment:
Qs: why we installed node-gyp? For  `prebuildify` node-gyp-build is required right?

Reply:
Yeah
https://github.com/prebuild/prebuildify-cross?tab=readme-ov-file#install

Node-gyp is shipped with npm anyway, but I really didn't like that we rely on the npm version.